docs/troubleshooting_and_faq/logs.md

```diff
@@ -40,55 +50,100 @@ log file followed by a number that represents rollover.
 - `validation.log`
   Contains a log of the validations performed by `mp-test` on the collection results. Validation logs are only generated locally.
 
+---
 ###  How do I add logs to my adapter?
 
+=== "Python Adapter Library"
 
-The template adapter defines a logger variable in the `adapter.py` file that configures all adapter logging using [adapter_logging](https://github.com/vmware/vmware-aria-operations-integration-sdk/blob/299ce55ac0b1e53dde9c1b4b3e217ab41df57299/lib/python/src/aria/ops/adapter_logging.py) from the Python SDK.
-The logger only needs to be configured once; to generate logs in other files, simply import the Python [logging](https://docs.python.org/3/library/logging.html) module. Eg.
+    The template adapter defines a logger variable in the `adapter.py` file that configures all adapter logging using 
+    [adapter_logging](https://github.com/vmware/vmware-aria-operations-integration-sdk/blob/299ce55ac0b1e53dde9c1b4b3e217ab41df57299/lib/python/src/aria/ops/adapter_logging.py) from the Python SDK.
+    The logger only needs to be configured once; to generate logs in other files, simply import the Python [logging](https://docs.python.org/3/library/logging.html) module. Eg.
 
-  ```python
-  import logging
+    ```python linenums="1" hl_lines="3"
+    import logging
 
-  logger = logging.getLogger(__name__)
+    logger = logging.getLogger(__name__)
 
-  def my_method():
-    logger.info("info log")
-    logger.warning("warning log")
-    logger.error("error log")
-    logger.debug("debug log")
-     ...
-  ```
+    def my_method():
+      logger.info("info log")
+      logger.warning("warning log")
+      logger.error("error log")
+      logger.debug("debug log")
+       ...
+    ```
 
+=== "Java Adapter Library"
 
+    Adapter logging is setup bia the [Get Logger](../references/java_project/java_lib/-integration-s-d-k-adapter-library/com.vmware.aria.operations/get-logger.html) method
```

Comment:
```suggestion
    Adapter logging is setup via the [Get Logger](../references/java_project/java_lib/-integration-s-d-k-adapter-library/com.vmware.aria.operations/get-logger.html) method
```

Reply:
It's actually setup by the `setupLogging(...)` method (same as Python Adapter). Generally the user won't have to call that, because it's part of the `main` method we provide, but we should still be accurate.